### DISCUSSION TITLE
Docker error [May 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2186] [Discord Threads]
> 🙋‍♂️ **untsvn**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### You may check docker log to see more insights  
#  
> 👨‍🚀 **wingkwong**  
#### one of the possible reasons s the connection problem to NC_DB  
#  
> 👨‍🚀 **wingkwong**  
#### mind to share your docker command with crendentials redacted?  
#  
> 🙋‍♂️ **untsvn**  
#### Legend <@495466997304262658> - thankyou! You prompt re the command reminded me I'd run from the command line previously when it worked not just done run image from the docker desktop thing. Going again now!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/976628227101581362)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
